mysql-test/suite/rpl/t/rpl_unpack_new_decimal_crash.test

```diff
@@ -0,0 +1,40 @@
+################################################################################
+# MDEV-33049 Assertion `marked_for_write_or_computed()' failed in bool
+#            Field_new_decimal::store_value(const my_decimal*, int*)
+#
+# In FULL_NODUP mode, the before image has all fields and the after image has
+# only updated fields. Crash happened when slave was unpacking a row event
+# if the new decimal field is null in the before image.
+################################################################################
+--source include/have_binlog_format_row.inc
+--source include/master-slave.inc
```

Comment:
Adding a new test adds a notable extra execution time. Also [de]initializing master-slave turns to be extra slow. If possible, please add this test somewhere in the existing test file related to the row-based replication.  `binlog_row_image` can be configured for a test case (inside a test file) just fine.

The good place would be one of `rpl/t/rpl_row_*.test` or `rpl/include/rpl_row_*.test` files

Reply:
Hi FooBarrior,
Thanks for your suggestions. I moved the test into rpl_row_img_eng_full_nodup.test.